server/src/main/kotlin/org/octopusden/octopus/dms/service/impl/ComponentServiceImpl.kt

```diff
@@ -224,6 +225,9 @@ class ComponentServiceImpl(
         componentsRegistryService.checkComponent(componentName)
         val (_, buildVersion) = normalizeComponentVersion(componentName, version)
         val componentVersionArtifactEntity = getOrElseThrow(componentName, buildVersion, artifactId)
+        if (componentVersionArtifactEntity.artifact.repositoryType == RepositoryType.DOCKER) {
+            throw UnsupportedOperationException("Docker artifacts can't be downloaded")
```

Comment:
The download of Docker artifacts is not supported.

Reply:
Fixed